### DISCUSSION TITLE
Why does this error keeps happening when i try to train a LoRA?

### DISCUSSION THREAD

**Discussion by Nojarosann** [kohya-ss/sd-scripts#446] [Q&A]
![Screenshot 2023-04-24 at 11-58-11 Google Colaboratory](https://user-images.githubusercontent.com/100730490/234134857-bb00c6c2-6a44-46d6-b607-4d0464206dec.png)

I got no idea why this keeps happening.

**Comment by kohya-ss** [comment]
If you could post the full text of the error, it might give us some insight. 

Also, it seems you are using Colab, if you are using someone else's notebook, please ask the author.

**Reply by Nojarosann** [reply]
![Screenshot 2023-04-25 at 11-29-51 Google Colaboratory](https://user-images.githubusercontent.com/100730490/234308641-60a9141b-4039-4bea-9ea9-74836a254172.png)
this is what appears above teh prvious error most of the time.

and oh... i actually thought this was the author's page...sorry i'll try to search for tehactual author's github i'm sorry for the bother
